tf2onnx/tfonnx.py

```diff
@@ -1222,11 +1232,6 @@ def minmax_op(ctx, node, name, args):
 
 
 def pack_op(ctx, node, name, args):
-    # in tf, "pack" can accept one input tensor which means doing nothing,
-    # so remove the node in ONNX
-    if len(node.inputs) == 1:
-        ctx.replace_all_inputs(ctx.get_nodes(), node.output[0], node.input[0])
-        return None
```

Comment:
@zhijxu-MS to take a look at this part.

Reply:
@zhijxu-MS  please help confirm this to unblock this PR. I think this should be a regression.

Reply:
original logic is right, my modification is not needed.

my fault, my issues in my real model is triggered by other problems.

Reply:
ok, got it. Thanks @zhijxu-MS for the double check.